exporters/ostream/src/log_exporter.cc

```diff
@@ -0,0 +1,90 @@
+#include "opentelemetry/exporters/ostream/log_exporter.h"
+
+#include <iostream>
+
+namespace nostd   = opentelemetry::nostd;
+namespace sdklogs = opentelemetry::sdk::logs;
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace logs
+{
+OStreamLogExporter::OStreamLogExporter(std::ostream &sout) noexcept : sout_(sout) {}
+
+sdklogs::ExportResult OStreamLogExporter::Export(
+    const nostd::span<std::shared_ptr<opentelemetry::logs::LogRecord>> &records) noexcept
+{
+  if (isShutdown_)
+  {
+    return sdklogs::ExportResult::kFailure;
+  }
+
+  for (auto &record : records)
+  {
+    // Convert trace, spanid, traceflags into string convertable representation
+    char trace_id[32] = {0};
```

Comment:
can 32 be made a constexpr or const, assuming it's just base 16 size of trace_id max digits.